lib/testing/src/lib/core/pages/data-table/data-table-item.ts

```diff
@@ -65,4 +65,10 @@ export class DataTableItem {
         await BrowserActions.click(row);
         await browser.actions().sendKeys(protractor.Key.ENTER).perform();
     }
+
+    async selectRowWithKeyboard(columnName: string, columnValue: string): Promise<void> {
```

Comment:
why do we need it?

Reply:
This method was copied from old datatable PO. I personally need it for deselecting a row in a ADW test. This method is also used is tests that are checking the selection with keyboard[but used from the other PO]